test/experiments_ClimaCore.jl

```diff
@@ -0,0 +1,80 @@
+# Experiments
+
+using Test
+using Pkg
+
+Pkg.activate(joinpath(@__DIR__,"../experiments/ClimaCore/tc4_atm-lnd-sfc-fluxes/"))
+
+Pkg.add(url="https://github.com/CliMA/ClimaAtmos.jl",rev="main" )
+Pkg.add(url="https://github.com/CliMA/SurfaceFluxes.jl",rev="sf_rc0.2" )
```

Comment:
We can add these to the `Project.toml`; we have the `Manifest.toml` checked in so the appropriate branches of these packages will be used.